docs/sql-guide/operators/operators-home.md

```diff
@@ -1,16 +1,68 @@
 ---
-title: SQL Operators
+title: Operators
 layout: default
 parent: SQL guide
-has_children: true
-nav_order: 4
-has_toc: false
 ---
 
 # SQL Operators
 
+SQL Operators are used in SELECT statements to perform mathematical and logical operations on operands.
+
+## Before you begin
+
+{% include /cloud/cloud-db-create-before-begin.md %}
+
+
 ## unary_op
 ![expr](/assets/images/sql-guide/unary_op.svg)
 
+| Operator | Description | Examples | Binary | Result |
+|:---:|---|---|---|---|
+| `!` | **Not** inverts all the bits of the operand. | `SELECT !5;` | `101` | `010` |
```

Comment:
Um, this was me trying to provide a way to interpret what's going on by showing another way to conceptualise the actual operation.

We were talking about bits for the operator, so that seemed to indicate binary, hence the "binary" column which represents the value 5.
Reversing all the bits becomes clearer by showing 5 represented in binary, and the result also in binary.
In hindsight I should've given another column for the decimal result too.

It's kind of like doing long-hand arithmetic when you move decimal points so you can divide by whole numbers instead of fractions -- https://www.mathsisfun.com/dividing-decimals.html

In this case, I suspect the column "binary" isn't a great description but it was all I could think of at the time.

Reply:
As suggested in a later comment, I've changed "binary" to "Binary representation" and "result" to "Binary result"

Should I add an actual result too to make it nice and clear? :D